documentation/release-notes/topics/version_5.md

```diff
@@ -1,5 +1,89 @@
 # Version 5
 
+## Hyperview 5.1 (Aug 22, 2025)
+
+This section covers significant changes and bug fixes in Hyperview 5.1.x since version 5.0.x
+
+:::{important}
+1. This release has changes and improvements to the Data Collector software. We strongly recommend updating
+   your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery
+   experience.
+
+2. Customers using Hyperview open-source tools such as hvcli/asset tool must update to the latest version.
+:::
+
+### New Licensed Feature: Accelerated Polling
+
+This feature allows administrators to specify sub five minute polling periods up to once every minute.
+
+:::{note}
+Please contact the Hyperview sales or support teams for more information.
+:::
+
+### Enhanced Feature: 2D and 3D Layout
+
+A new view filter was added to the Location layout. This will allow Hyperview users to filter and report on various ad-hoc data in the system.
+
+For example, If you have a custom property that classifies assets by network location, such DMZ, Production, Test and you have that data captured in Hyperview. You will be able to create a report that would quickly visualize the location and distribution of assets that carry these properties.
+
+Please refer to the Hyperview product documentation and user guide for more information.
+
+### Enhanced Feature: Asset Dashboard
+
+This new widget (Associations) displays asset association to business entities, linked sensors, power providers and network connections and circuits. It provides one view to see these dependencies and connections in one place.
+
+### Enhanced Feature: Catalog
+
+This features improves the usability of the catalog by allowing users that want to customize existing models to easily clone existing data and perform the needed customization, for example, chance product images.
+
+### Enhanced Feature: Chart time range selector
+
+The time range selector for charts has been improved to add support for local time based charts where possible. Previously, The time range selector allowed only for UTC based time ranges.
+
+### Enhanced Feature: Data Grids
+
+Grid double click action has been improved to be more consistent and available on more more data grids. This will improve the navigation experience within the product.
+
+Grids now have 2 modes, an adaptive mode, which is the existing default, where rows would compress to should applicable data depending on the width of the screen. And a non-adaptive mode, where grids will enable horizontal scrolling to show columns extra column. The mode is activated by a button next to the grid column selector.
```

Comment:
I think: "scrolling to show columns extra column."
should be: "scrolling to show extra columns." or something like that.